src/features/venuemap/components/MapPinWithCounter/MapPinWithCounter.tsx

```diff
@@ -5,28 +5,33 @@ import { MapPin } from 'ui/svg/icons/MapPin'
 import { getShadow, getSpacing, Typo } from 'ui/theme'
 
 type Props = {
-  count?: number
+  count: number
 }
 
 const NUMBER_LEFT_POSITION = 15
-const NUMBER_TOP_POSITION = -10
-
-export const VenueMapPin: FunctionComponent<Props> = ({ count }) => {
-  const shouldDisplayCounter = count && count > 1
+const NUMBER_TOP_POSITION = -2
+const MAPPIN_SIZE = 32
```

Comment:
```suggestion
const MAP_PIN_SIZE = 32
```
C'est en 2 mots non ?

Reply:
D'ailleurs, le MAP_PIN_SIZE n'est pas fourni à MapPin, c'est bizarre non ?